### PR TITLE
javascript2-week1/srushti

### PR DESCRIPTION
**Pull request by srushtisalke**
Review my  homework.

### CODE REVIEW

File: 3_javascript/javascript2/week1/script.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+//Find and count the Danish letters
+
+function countDanishLetters(inputString) {
```
Comment (shpomp) [inline 3_javascript/javascript2/week1/script.js:3]: Very nice and clear! 👍🏽

File: 3_javascript/javascript2/week1/script.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    return spiritAnimals[randomIndex];
+  }
+});
+
+const newAnimalButton = document.getElementById("newAnimalButton");
+
+newAnimalButton.addEventListener("click", function () {
+  generateSpiritAnimal();
+});
+
```
Comment (shpomp) [inline 3_javascript/javascript2/week1/script.js:111]: Once again, very nice and well done :)

**Review by shpomp — APPROVED** [review]
Overall really good job! 

However, there is a homework from JS1 here that should not be there, which would be necessary to avoid 🙏🏽

**Comment by srushtisalke** [comment]
Removed  few part of JS script as it was related to JS2 week2.
Deleted JS1 week4 homework files as it was mistakenly committed.

**Pull request closed without merging**